src/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,21 @@
+import data from "../Data/stretch-goal.json"

```

Comment:
Folders and files (except for React components) should be named in kebab-case, lowercase

Reply:
I only managed to change the name of the sidebar file. But I will remember this and avoid the mistake in the next project.